bazel/emscripten_toolchain/env.bat

```diff
@@ -3,3 +3,7 @@
 set ROOT_DIR=%CD%
 set EMSCRIPTEN=%ROOT_DIR%\%EM_BIN_PATH%\emscripten
 set EM_CONFIG=%ROOT_DIR%\%EM_CONFIG_PATH%
+
+:: Ensure PYTHONSAFEPATH is not set so that modules in the same
+:: directory as emcc.py and emar.py etc. can be found via sys.path
```

Comment:
Maybe mention that this is needed because py_binary sets this?

Alternatively, perhaps there is a more canonical way to express this in a `py_binary`?  i.e. perhaps there is an explicit way to add the emscripten root the `PYTHONPATH`?

Reply:
`py_binary` is the more idiomatic solution. Although I am blocked on this at the moment due to a bug in `python_rules`.